### DISCUSSION TITLE
Unable to pass the verification for GitHub Student Program

### DISCUSSION THREAD

**Discussion by paramjeet-dev** [community/community#134867] [GitHub Education]
### Hi everyone,

Hello everyone

I'm just tired of the boring process for applying for the education program. They just keep on rejecting my application stating the same reason again and again.

> Attendees of this school must verify and use their academic email address. When you have verified your school's academic email address on your account, ensure that you select the school when prompted on the application page.

I have tried multiple ways like my gmail id, typing The school don't provide me school mail, Not Applicable etc but none of them seems to work. Can Anyone of you can help me get away with this thing?

**Comment by eliancheng** [comment]
I understand your frustration! If your school doesn't provide an academic email, try contacting the program's support team directly and explain your situation. They may offer an alternative verification method or provide guidance on how to proceed with your application. Good luck!

**Reply by paramjeet-dev** [reply]
Can't reach to their support. They have stopped creation of tickets for education program issues.
